doc/reference/configuration/cfg_logging.rst

```diff
@@ -11,7 +15,7 @@
 .. confval:: log_level
 
     Since version 1.6.2.
-    What level of detail the :ref:`log <admin-logs>` will have. There are seven levels:
+    Specifies the level of detail the :ref:`log <admin-logs>` has. There are seven levels:
```

Comment:
Maybe just drop the verb?
"The level of detail..."

Reply:
I'd keep `Specifies` here as it shows that the property can be get and set. What do you think?